tests/gtest/aviftest_helpers.cc

```diff
@@ -501,5 +501,85 @@ avifIO* AvifIOCreateLimitedReader(avifIO* underlyingIO, uint64_t clamp) {
 
 //------------------------------------------------------------------------------
 
+template <typename SampleType>
+bool IsOpaque(const avifImage* image) {
+  if (!image->alphaPlane) {
+    return true;
+  }
+
+  const SampleType opaque_alpha = (1 << image->depth) - 1;
+  for (uint32_t j = 0; j < image->height; ++j) {
+    const SampleType* const row = reinterpret_cast<SampleType*>(
+        image->alphaPlane + j * image->alphaRowBytes);
+    for (uint32_t i = 0; i < image->width; ++i) {
+      if (row[i] != opaque_alpha) {
+        assert(row[i] < opaque_alpha);
+        return false;
+      }
+    }
+  }
+  return true;
+}
+
+bool IsOpaque(const avifImage* image) {
+  return avifImageUsesU16(image) ? IsOpaque<uint16_t>(image)
+                                 : IsOpaque<uint8_t>(image);
+}
```

Comment:
`avifImageIsOpaque()` can be used instead

Reply:
Fixed, thanks.